<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weather</class>
 <widget class="QWidget" name="Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>339</y>
     <width>199</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>299</y>
     <width>199</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>83</y>
     <width>199</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_wind">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>379</y>
     <width>199</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_weather_pic">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>145</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_week">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>199</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string>date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
